rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/helm/HelmBakeManifestRequest.java

```diff
@@ -8,4 +8,5 @@
 @EqualsAndHashCode(callSuper = true)
 public class HelmBakeManifestRequest extends BakeManifestRequest {
     String namespace;
+    String outputArtifactName;
```

Comment:
I think you can put this in `BakeManifestRequest`, since this isn't specific to helm.

Reply:
@lwander fixed